src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3584,3 +3664,308 @@ map<long, map<long, unordered_set<long>>> calculateEgoNet(string graphID, string
+                            JasmineGraphHashMapCentralStore centralDB,
+                            std::vector<string> workerSockets) {
+
+    map<long, unordered_set<long>> centralGraphMap = centralDB.getUnderlyingHashMap();
+    map<long, unordered_set<long>> localGraphMap = localDB.getUnderlyingHashMap();
+    map<long, unordered_set<long>>::iterator localGraphMapIterator;
+    map<long, unordered_set<long>>::iterator centralGraphMapIterator;
+
+    std::vector<long> vertexVector;
+    for (localGraphMapIterator = localGraphMap.begin();
+         localGraphMapIterator != localGraphMap.end(); ++localGraphMapIterator) {
+        long startVid = localGraphMapIterator->first;
+        unordered_set<long> endVidSet = localGraphMapIterator->second;
+
+        for (auto itr = endVidSet.begin(); itr != endVidSet.end(); ++itr) {
+            if (localGraphMap.find(*itr) == localGraphMap.end()) {
+                unordered_set<long> valueSet;
+                localGraphMap.insert(std::make_pair(*itr, valueSet));
+            }
+        }
+    }
+
+    map<long, long> localOutDegreeDistMap = localDB.getOutDegreeDistributionHashMap();
+
+    long partitionVertexCount = localGraphMap.size();
+    long worldOnlyVertexCount = atol(graphVertexCount.c_str()) - partitionVertexCount;
+
+    double damp = 1 - alpha;
+    int M = partitionVertexCount + 1;
+
+    long adjacencyIndex[M];
+    int counter = 0;
+
+    for (localGraphMapIterator = localGraphMap.begin();
+         localGraphMapIterator != localGraphMap.end(); ++localGraphMapIterator) {
+        long startVid = localGraphMapIterator->first;
+
+        adjacencyIndex[counter] = startVid;
+        counter++;
+    }
+
+    adjacencyIndex[partitionVertexCount] = -1;
+
+    map<long, float> hMapAuthorityFlowWorldToLocal = getAuthorityScoresWorldToLocal(graphID, partitionID, serverPort,
+                                                                                    graphVertexCount, localDB,
+                                                                                    centralDB,
+                                                                                    localGraphMap, workerSockets,
+                                                                                    worldOnlyVertexCount);
+
+    map<long, unordered_set<long>> hMapFromVerticesWorldToLocalFLow = getEdgesWorldToLocal(graphID, partitionID,
+                                                                                           serverPort, graphVertexCount,
+                                                                                           localDB, centralDB,
+                                                                                           localGraphMap,
+                                                                                           workerSockets);
+
+    long entireGraphSize = atol(graphVertexCount.c_str());
+    int T = 2;
```

Comment:
Why is the `T` set to 2 here? Better add a comment explaining the reason.

Reply:
This was a redundant value. Hence removed

Reply:
Ok